bindings/node/lib/clientEncryption.js

```diff
@@ -66,6 +66,7 @@ module.exports = function(modules) {
       this._client = client;
```

Comment:
I can't comment above because of the way Github structures diffs but should we add the option, `tlsOptions`, to the doc string above?  Looks like the other options are included in the doc string.

Reply:
Added.